### ISSUE TITLE
Packet structures are out of date

### ISSUE THREAD

**Issue by TadusPro** [thomas-crane/realmlib-net#5]
https://i.imgur.com/SX52j4Z.png

**Comment by crinny** [comment]
**TL;DR** In order for it to work you need to add an empty string at the end of HELLO packet:
```javascript
writer.writeString("");
```

The only difference I could find is field `previousConnectionGuid`
![image](https://user-images.githubusercontent.com/42045258/80379253-81b08780-88a6-11ea-9d95-546287363f77.png)
It is being set at line 2215 of `kabam\rotmg\messaging\impl\GameServerConnectionConcrete.as` (Flash client of course)
```java
private function onMapInfo(param1:MapInfo) : void
{
   var _loc2_:String = null;
   var _loc3_:String = null;
   for each(_loc2_ in param1.clientXML_)
   {
      this.parseXML(_loc2_);
   }
   for each(_loc3_ in param1.extraXML_)
   {
      this.parseXML(_loc3_);
   }
   changeMapSignal.dispatch();
   this.closeDialogs.dispatch();
   gs_.applyMapInfo(param1);
   this.rand_ = new Random(param1.fp_);
   if(createCharacter_)
   {
      this.create();
   }
   else
   {
      this.load();
   }
   connectionGuid = param1.connectionGuid_;  // connectionGuid is being used in HELLO
}
```

Default value is "". Setting it to empty string leads to client getting MAPINFO and everything going good.

P.S. Setting is to whitespace (or anything invalid I assume) leads to some funny behavior:
```python
[0] (0, 'Invalid connection guid, you want to get banned?')  # scary
```

**Comment by thomas-crane** [comment]
Closed in a8440d6cfd1c21fdb84929ee6264612bc6837c99
